pkg/api/api.go

```diff
@@ -78,6 +86,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 func (cfg *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.StringVar(&cfg.AlertmanagerHTTPPrefix, prefix+"http.alertmanager-http-prefix", "/alertmanager", "HTTP URL path under which the Alertmanager ui and api will be served.")
 	f.StringVar(&cfg.PrometheusHTTPPrefix, prefix+"http.prometheus-http-prefix", "/prometheus", "HTTP URL path under which the Prometheus api will be served.")
+	f.StringVar(&cfg.corsRegexString, prefix+"server.cors-origin", ".*", "Access-Control-Allow-Origin response header.")
```

Comment:
Do we need to mention the additional ancors we added? In case someone tries to add anchors themselves.

Reply:
sorry, I don't understand. what anchors are you referring to?

Reply:
What I meant is that `compileCORSRegexString` will add an additional anchor and we probably need to mention it in the flag description.
I think you can maybe just copy over what Prometheus has https://github.com/prometheus/prometheus/blob/main/cmd/prometheus/main.go#L296 and we should be ok